fluentd/generate_throttle_configs.rb

```diff
@@ -2,16 +2,35 @@
 require 'date'
 require 'logger'
 
+log_level = ENV['LOG_LEVEL'] || 'INFO'
```

Comment:
doesn't it make sense to have this default at 'WARN' given that's what we do for the rescue below?